app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -226,6 +243,7 @@ class HistoryFragment : LibraryPageFragment<HistoryItem>(), UserInteractionHandl
                     }
 
                     dialog.dismiss()
+                    showSnackBar(view!!, getString(R.string.preferences_delete_browsing_data_snackbar))
```

Comment:
Same nit as above - can we move this into the `launch(Main)` above to tie this to the view lifecycle?

Reply:
Agreed, moved it inside the scope.